torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2794,21 +2805,25 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                                 device_type='cuda', dtypes=[torch.cdouble], active_if=IS_WINDOWS),
                    )),
     OpInfo('addmm',
-           dtypes=floating_types(),
+           dtypes=floating_and_complex_types_and(torch.float16),
```

Comment:
Add a comment here that this addmm is for when alpha and beta are not both equal to 1 (and that alpha=beta=1 is tested in the following opinfo because that special case will trigger addmm being decomposed by a jit pass)